### ISSUE TITLE
Tests are run twice

### ISSUE THREAD

**Issue by rubenvereecken** [jasmine-contrib/grunt-jasmine-node#68]
My extremely simple specs folder containg a single test with a single assertion is ran twice. The `jasmine_node` object is simply `{specFolders: ['specs']}`, though I've used a number of additional but equally unhelpful options in an attempt to fix it.

Any ideas?

**Comment by florin-ardelian-moby** [comment]
Yes. Don't specify a specFolders option. Look at the source code of this grunt task (https://github.com/jasmine-contrib/grunt-jasmine-node/blob/master/tasks/jasmine-node-task.js) and you will see that it will automatically append `.` (the current directory) to the list of specFolders.

What I would do if I were you (and what I actually did a year ago when I used grunt-jasmine-node) would be to copy the source code for this grunt task from the link I just gave you, fix the bugs, use it to create your own grunt task, and stop using the project from this github repo because it's not maintained.
